### DISCUSSION TITLE
UnoCSS Grid Areas plugin.

### DISCUSSION THREAD

**Discussion by chz** [unocss/unocss#3006] [Q&A]
I've created UnoCSS plugin called Grid Areas based on [Grid Areas for Tailwind CSS](https://savvywombat.com.au/tailwind-css/grid-areas). How can i add this plugin to community plugins in unocss? 

Link: https://github.com/chz/unocss-preset-grid-areas
NPM: https://www.npmjs.com/package/unocss-preset-grid-areas

**Comment by chz** [comment]
Also please check and PR welcome.

**Reply by userquin** [reply]
You should review the package exports in your preset.

Can tou show some example?

**Reply by userquin** [reply]
It is not in npm registry yet.

**Reply by chz** [reply]
@userquin you can check demo here: https://stackblitz.com/edit/unocss-unocss-evdgbq?file=pages%2Findex.vue or play with it :). This demo for Nuxt 2. But it will be also work on Nuxt 2.

Clear your npm cache then check locally maybe you have cache.
